kafka-ui-api/src/main/java/com/provectus/kafka/ui/controller/KafkaConnectController.java

```diff
@@ -71,6 +72,18 @@
     return Mono.just(ResponseEntity.ok(kafkaConnectService.getAllConnectors(clusterName)));
   }
 
+  @Override
+  public Mono<ResponseEntity<Flux<FullConnectorInfo>>> getFilteredConnectors(
```

Comment:
What's the reason to keep `Flux` inside the `Mono`?

Reply:
Honestly @Tereshin I tried to stick to the same way similar methods do it.